autosklearn/ensemble_builder.py

```diff
@@ -314,7 +319,8 @@ def read_ensemble_preds(self):
                     "seed": _seed,
                     "num_run": _num_run,
                     "budget": _budget,
-                    "deleted": False,
+                    "is_new": True,
```

Comment:
Hey, it seems like the `is_new` key is never used, is that correct?

Reply:
Ops, absolutely correct. It was an old idea. Forgot to delete.